core/util/src/main/java/orca/util/ScriptExecutor.java

```diff
@@ -65,6 +102,12 @@ public ExecutionResult execute() throws Exception
             proc = rt.exec(cmdArray);
         }
 
+        ReadStream stdinRstr = new ReadStream("stdin", proc.getInputStream(), logger);
+        ReadStream errRstr = new ReadStream("stderr", proc.getErrorStream(), logger);
+        
+        stdinRstr.start();
+        errRstr.start();
+        /*
```

Comment:
OK I'll delete. I noticed other implementations of the same executor. Seems like maybe this has come up before. I don't know why this code stopped working or why the new one works. They are mostly equivalent, except new code is multi-threaded.